### DISCUSSION TITLE
API Integration with Cisco Prime Infrastructure

### DISCUSSION THREAD

**Discussion by RyanMountain** [netbox-community/netbox#10433] [Q&A] [answered]
Hi,

I'll get right to the point and sorry in advance if I use incorrect terminology, all quite new to me.

When I create a site within NetBox, I would like this to then go off and automatically create the location within Cisco Prime Infrastructure. Is this something I can configure on NetBox and if so where? Or do I need make an API call from Prime to get the site?

Thanks in advance.

**Comment by RyanMountain** [comment, marked as answer]
Not to worry, I believe Webhooks are my Answer to this?

**Comment by jcollie** [comment]
Webhooks would be part of it. You'll still need something running externally to take the info from NetBox and use Prime API calls to create the sites.
